test/tests/functional/pbs_node_buckets.py

```diff
@@ -187,7 +187,7 @@ def test_multi_bucket2(self):
             self.assertTrue(self.colors[i] in
                             n[0]['resources_available.color'])
 
-    @timeout(450)
```

Comment:
did we check if 450 seconds made more sense than the default of 600?

Reply:
This was added when the default timeout used to be 180s. I have gone through the tests and I didn't find anything that would get affected due to the change in the timeout value(450s to 600s). If the tests passes it will exit even if it hasn't hit timeout value.